### DISCUSSION TITLE
white Image flash bevor first frame while looping

### DISCUSSION THREAD

**Discussion by Rueder** [PaulleDemon/tkVideoPlayer#22] [General]
Hi,
im using the following code to loop the Video.
every time the Video starts i get a short white Image popping up
the duration of the "white Frame" is way shorter than the Frame time of the Video.
The Video is only 3 seconds long and permanetly looping and it is clearly to se every time.
Same "white Frame" no matter which Video i use ore on which timestamp i start same Problem.
```
def loopVideo(event):
    videoplayer.play()

videoplayer.bind('<<Ended>>', loopVideo)
```
maby you can help me.
If it is not fixable thats ok. But it is slightly annoying.

**Comment by PaulleDemon** [comment]
Hey, @Rueder, it might be the white background of the label on which the video is being played. you can try adding color by `video_player.configure(background='black')`. Let me know if this helps

**Reply by Rueder** [reply]
Thanks for the fast replay.
yes indeed it changes the color of the flashing Frame, but i really hoped i can get rid of the flash somehow.
(i already dried setting the `videoplayer.configure(image="...")` to the first frame of the video. Didn't work)
But if not its at least a little better in grey.
Thanks
